### DISCUSSION TITLE
Brew's pip3 installs libraries and binaries in `~/Library/Python/3.10/bin` when using the `--user` flag

### DISCUSSION THREAD

**Discussion by carlescufi** [Homebrew/discussions#3637] [Getting started] [answered]
It seems that invoking:

`/opt/homebrew/bin/pip3 install --user <package>` installs the "executable" (i.e. the small python script) in `~/Library/Python/3.10/bin` instead of `/opt/homebrew/bin/`. This also applies to the packages themselves.

This is on an M2 Macbook Air.

Thanks!

**Comment by SMillerDev** [comment, marked as answer]
Yes, pip installs in `~/Library` if you pass `--user`. That's expected behavior.

**Reply by carlescufi** [reply]
Understood, thanks for clarifying.
